src/components/Dashboard/SortList/SortList.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { Component } from 'react';
+import {
+  Dropdown,
+  ButtonGroup,
+  Button,
+} from 'react-bootstrap';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import * as sortListActions from '../../../actions/sortListActions';
+import './SortList.scss';
+import { ReactComponent as SortIcon } from './sort.svg';
+import { ReactComponent as SortIconAsc } from './sort-asc.svg';
+import { ReactComponent as SortIconDesc } from './sort-desc.svg';
+
+class SortList extends Component {
+  state = {};
```

Comment:
Why the state should be outside the costructor?

Reply:
oops. This variable is not needed. I forgot to remove it.